### PR TITLE
refactor(ui): use single file for duplicated markdowns

### PR DESCRIPTION
**Pull request by wlee221**
*Issue #, if available:*

*Description of changes:* We have common markdowns that are used across many ui pages. This includes `yarn install` instructions, `main.js` setup for vue, and `app.module.ts` setup for angular. They are copy-pasted across our fragments and thus hard to maintain. This PR centralize them at ~/docs/ui/fragments so that there is only one file to manage for each of them.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/ui/fragments/angular/configure-module.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+```js
```
Comment (tjleing) [inline docs/ui/fragments/angular/configure-module.md:1]: Why not put `_app.module.ts_` at the top here so you don't have to duplicate it as well? (similar for `main.js` above vue/configure-app.md)

**Review by tjleing — APPROVED** [review]
Looks good!  Thank you @wlee221 😄

File: docs/ui/fragments/angular/configure-module.md (hunk 1 above)
Comment (wlee221) [reply docs/ui/fragments/angular/configure-module.md:1]: Good point, I will do that.

File: docs/ui/auth/fragments/ionic/auth-state-management.md [hunk 2]
```diff
@@ -1,22 +1,4 @@
...
-
-@NgModule({
-  declarations: [AppComponent],
-  imports: [AmplifyUIAngularModule, BrowserModule],
-  providers: [],
-  bootstrap: [AppComponent],
-})
-export class AppModule {}
-```
+<inline-fragment src="~/ui/fragments/angular/configure-module.md"></inline-fragment>
```
[the remark this answers is not in the document]
Comment (wlee221) [reply docs/ui/auth/fragments/ionic/auth-state-management.md:1]: Yeah, that's why I didn't made separate entry for ionic, but we can revisit this later

**Review by tjleing — APPROVED** [review]

**Comment by wlee221** [comment]
I found that migration guides can share the same file too, so I'm working on that. Sorry I'm making you review this multiple times @tjleing 😢

**Review by tjleing — APPROVED** [review]
Good find!

**Comment by wlee221** [comment]
Thanks @tjleing!

**Pull request merged**